internal/api/policy.go

```diff
@@ -65,7 +65,7 @@ func (s *Server) verifyPolicySignature(policy types.PluginPolicy, update bool) b
 		return false
 	}
 
-	isVerified, err := sigutil.VerifySignature(policy.PublicKey, policy.ChainCodeHex, msgBytes, signatureBytes)
+	isVerified, err := sigutil.VerifySignature(policy.PublicKey, policy.ChainCodeHex, policy.DerivePath, msgBytes, signatureBytes)
```

Comment:
Please check with @RaghavSood , we removed derivePath from policy

Reply:
ref: https://github.com/vultisig/verifier/pull/40 ,  you are overriding the changes in that PR

Reply:
fixed